ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -123,6 +122,16 @@ export const ProfilePage = withUserProfile()(class extends React.Component<
       }[v] || validate.prettify(v))
     });
 
+    // render a value as US currency: $255.37
```

Comment:
Would be good to also indicate what the input format is. I've worked in systems where currency was tracked as cents with integers.

Reply:
Good point.  Added.